src/Arduino_ConnectionHandler.h

```diff
@@ -22,8 +22,8 @@
    INCLUDES
  ******************************************************************************/
 
-#include <Client.h>
-#include <Udp.h>
+//#include <Client.h>
```

Comment:
Ideally there should never be out-commented code within a source file. Either you delete it or you leave it entirely. With commented out code a future reader wonders: Why is this commented out? Did someone try something out and forgot to remove the `//` ? Is this a fix of some sort?

Reply:
Fixed